employee managment/db.js

```diff
@@ -0,0 +1,21 @@
+// db.js
+const { Sequelize } = require('sequelize');
+const { DB_USERNAME, DB_PASSWORD, DB_HOST, DB_PORT, DB_NAME } = require('./config');
+
+const sequelize = new Sequelize(DB_NAME, DB_USERNAME, DB_PASSWORD, {
+  host: DB_HOST,
+  port: DB_PORT,
+  dialect: 'mariadb',
+  logging: console.log,
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider disabling SQL logging in production</b></div>
<div id="fix">

Consider disabling SQL query logging in production environments. The current setup with `logging: console.log` may expose sensitive information and impact performance.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -8,7 +8,7 @@
    dialect: 'mariadb',
 -  logging: console.log,
 +  logging: process.env.NODE_ENV === 'development' ? console.log : false,
  });
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/rishabhbohra3/Bito-CRA-Issues-Dataset/pull/6#issuecomment-2513272175>#4c9a2d</a></i></small>
</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged